snapcraft/tests/unit/test_errors.py

```diff
@@ -58,9 +57,8 @@ class ErrorFormattingTestCase(unit.TestCase):
                 "The 'test-step' step of 'test-part' is out of date:\n"
                 "The 'test-property1' and 'test-property2' part properties "
                 "appear to have changed.\n"
-                "To continue, please clean that part's "
-                "'test-step' step by running:\n"
-                "snapcraft clean test-part -s test-step\n")}),
+                "To clean that part's 'test-step', run "
```

Comment:
This feels a little... jarring to me. Our guidance is "We have a problem. Here's why it's a problem. Here's how you fix it." What we're moving to here seems to be "We have a problem. Here's why it's a problem. Run this command to do X."

It seems a small difference, but there's a disconnect I can't quite explain between "Here's why it's a problem" and "Run this command." It doesn't flow, it seems like we're missing the "here's how you fix it" part, and just throwing a magical command on there. Is that just me?

Reply:
So, like explaining what will happen in english and then give the command that should be run?
I'm not really sure I'm getting your comment, so maybe a suggestion of an error message that follows your proposed style would be useful.